### ISSUE TITLE
newline character in Answer/Pinboard description breaking --export

### ISSUE THREAD

**Issue by boonhapus** [thoughtspot/cs_tools#18]
### First Stop

- [X] I have read and agreed to the ThoughtSpot Dev Tools EULA.
- [X] I understand that if I have a critical issue with ThoughtSpot, I should reach out to my account team instead of cs_tools.
- [X] I searched the cs_tools documentation, with the integrated search.

### Platform Configuration

```
        [PLATFORM DETAILS]
        system: Windows (detail: Windows-10-10.0.19041-SP0)
        python: 3.8.5
        datetime: 2021-12-08 17:28:52-0600
        cs_tools: 1.2.0
```

### Example Command

`cs_tools tools searchable-content gather --export /path/to/my/directory --config my-config`

### Description

> When extracting data with the searchable-content tool, there is a system view that has double-quotes in the description because the actual description has embedded carriage returns.
> 
> When the csv file gets created, the data ends up with carriage returns in the csv file.
> 
> GUID: 235f049a-e201-4e67-9bf5-14fcf3f264cf	
> Name: TS: Daily Row Count External Table [Deprecated]	
> Description: Aggregate Worksheet to keep track of following information for external table on a daily basis.
> 
> Does the --export function have any capability to look for double-quoted fields and ensure they write out on a single exported line?
> 
> I have not tested this on another instance - the screenshot below is extracting data directly from the Logitech cluster.
> And, the attached csv file shows 2 lines of output - the 3 lines related to the view above, and then one additional row of data.

**Comment by boonhapus** [comment]
https://github.com/thoughtspot/cs_tools/blob/5e2cdbed3837781d1bae1b2d25b9304264fbedf3/cs_tools/tools/common.py#L299-L320

Simply need to augment `common.to_csv` to accept proper quoting. At least `L315`.

More appropriate [format parameters](https://docs.python.org/3.6/library/csv.html#csv-fmt-params) are required.

**Comment by boonhapus** [comment]
🐛 Fixed in [v1.3.0](https://github.com/thoughtspot/cs_tools/releases/tag/v1.3.0).
